tools/tools.go

```diff
@@ -0,0 +1,11 @@
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/axw/gocov/gocov"
+	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
+	_ "github.com/matm/gocov-html"
+	_ "github.com/vasi-stripe/gogroup/cmd/gogroup"
+	_ "golang.org/x/tools/cmd/cover"
+)
```

Comment:
I don't think I've understood this part I'm sorry! Why do we need these libs?

Reply:
These libs are a part of CI checks, they're not necessary, but they're useful.
github.com/axw/gocov/gocov
github.com/matm/gocov-html
golang.org/x/tools/cmd/cover
These for coverage report.

github.com/vasi-stripe/gogroup/cmd/gogroup
This for ordering imports

github.com/golangci/golangci-lint/cmd/golangci-lint
This for linting

This is an approach of having the same version of the toolset to have reproducible builds.
https://marcofranssen.nl/manage-go-tools-via-go-modules/
https://github.com/nezorflame/golang-talks/blob/master/2020/10/modules/go_modules_in_production.pdf